Update documentation for v2.2 (#162)
ServiceNow/azimuth: docs/docs/user-guide/exploration-space/utterances-table.md

### Smart Tags

> The table shows smart tag families along with the specific smart tag values. For more

Reviewer: Should we mention users have to hover?
```suggestion
The table shows smart tag families, as well as the specific smart tag values on hover. For more
```
